recipe/meta.yaml

```
@@ -143,7 +143,7 @@ outputs:
     build:
       run_exports:
         - {{ pin_subpackage('mkl') }}
-        - {{ pin_subpackage('blas', exact=True) }}
+        - {{ pin_compatible('blas') }}
```

```suggestion
        - blas=*=mkl
```
To be more explicite.